doc/src/reference/py.rst

```diff
@@ -126,6 +131,37 @@ target
     :rtype: target
 
 
+module
+------
+.. py:method:: print()
+
+    This prints the contents of the module as list of instructions
```

Comment:
```suggestion
    Prints the contents of the module as list of instructions
```

Reply:
In general, can we keep the conventions in this file consistent? For example, having a period at the end of each statement. If there are other lines in the file where this change needs to be made, feel free to modify them as well.

Reply:
yes, Thanks. I made changes for the method's and functions' descriptions. but for param and rtype many of the statements are incomplete or short statement, i haven't not modified them.